src/bundle.ts

```diff
@@ -83,14 +122,16 @@ export class BuildBundler extends Transform {
   }
 
   async _buildBundles(): Promise<Map<string, string>> {
-    let strategy: BundleStrategy;
-    if (this.config.shell) {
+    let strategy = this._strategy;
```

Comment:
```typescript
const strategy = this._strategy ||
        this.config.shell && generateShellMergeStrategy(urlFromPath(
                                 this.config.root, this.config.shell));
```

Went with the above over ternary because even shorter.